programs/degen-pools/src/pools.rs

```diff
@@ -80,7 +80,7 @@ pub fn set_winning_option(ctx: Context<UpdatePool>, winning_option: Pubkey) -> R
 pub struct UpdatePool<'info> {
     #[account(mut)]
     pub pool_account: Account<'info, Pool>,
-    #[account(mut)]
+    #[account(mut, address = AUTHORITY_PUBKEY)]
```

Comment:
This crucial code was missing to avoid letting non admin users pause pools/set winners

Reply:
Jeeeeesus 😂 I can't believe none of us noticed this. Thank god we didn't open source it.

gg on the save, capn 🫡